content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -94,7 +94,15 @@ To generate Prisma Client with the Data Proxy, call `prisma generate` with the o
 npx prisma generate --data-proxy
 ```
 
-When you use `--data-proxy`, you get a very lightweight Prisma Client runtime of approximately 250kb. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
+Alternatively, you can set the environment variable `PRISMA_GENERATE_DATAPROXY=true` to generate a Prisma Client with Data Proxy when a normal `prisma generate` command is executed.
+
+<Admonition type="warning">
+
+When deploying to [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) you _need_ to define `PRISMA_GENERATE_DATAPROXY=true` [as an environment variable in your Vercel project settings](https://vercel.com/docs/concepts/projects/environment-variables) for the correct Prisma Client for Data Proxy to be generated.
+
+</Admonition>
+
+When you use `--data-proxy` or `PRISMA_CLI_GENERATE_DATAPROXY=true`, you get a very lightweight Prisma Client runtime of approximately 250kb. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
```

Comment:
```suggestion
When you use `--data-proxy` or `PRISMA_GENERATE_DATAPROXY=true`, you get a very lightweight Prisma Client runtime of approximately 250kb. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
```

Reply:
We need to correct this here. 250kb is only true for `@prisma/client/edge`, being engine-less that would be true for both.

Reply:
I felt this exact mention was pretty weird anyway and had though about removing it in https://github.com/prisma/docs/pull/3275. What do you think @andrew-walford-prisma?

Reply:
Removed "of approximately 250kb" for now as it is a) incorrect and b) weirdly specific for something that really depends on the schema size etc as well.

Reply:
Persisted in https://github.com/prisma/docs/issues/3285